src/Nethermind/Nethermind.Evm.Test/Eip7516BlobBaseFeeTests.cs

```diff
@@ -41,7 +42,11 @@ public void Blob_Base_fee_opcode_should_return_expected_results(bool eip7516Enab
         TestAllTracerWithOutput tracer = CreateTracer();
         _processor.Execute(transaction, block.Header, tracer);
 
-        _ = BlobGasCalculator.TryCalculateBlobGasPricePerUnit(excessBlobGas, out UInt256 expectedGasPrice);
+        UInt256 expectedGasPrice = 1; // default value of gas price when excess is 0
```

Comment:
Do you mean `Eip4844Constants.MinBlobGasPrice`?

Reply:
removed